contracts/SubdomainRegistrar.sol

```diff
@@ -45,10 +47,22 @@ contract SubdomainRegistrar is RegistrarInterface {
     uint referralFeePPM;
   }
 
+  mapping (bytes32 => address) deedOwners;
```

Comment:
There's no need to own this - we should just maintain a single `owner` property in `Domain`. In order to allow precommitment to a new custodian, we can have a second "transferAddress", that is the only address we permit transfer to once there's a new registrar, and can't be changed once it's set.